cirq-core/cirq/protocols/json_serialization_test.py

```diff
@@ -951,11 +951,16 @@ def test_basic_time_assertions():
 def test_datetime():
     naive_dt = datetime.datetime.now()
 
-    with pytest.raises(TypeError):
-        cirq.to_json(naive_dt)
+    re_naive_dt = cirq.read_json(json_text=cirq.to_json(naive_dt))
+    assert re_naive_dt != naive_dt, 'loads in with timezone'
```

Comment:
Maybe also add an assertion that the timestamp is unchanged:
```python
assert re_naive_dt.timestamp() == naive_dt.timestamp()
```

Reply:
added, thanks